operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml

```diff
@@ -113,13 +187,18 @@ spec:
     * Integrated with OpenShift Developer Console
 
     ## Installation
+
     _Red Hat OpenShift Pipelines Operator_ gets installed into a single namespace (openshift-operators) which would then install _Red Hat OpenShift Pipelines_ into the openshift-pipelines namespace. _Red Hat OpenShift Pipelines_ is however cluster-wide and can run pipelines created in any namespace.
+
     ### Components
-    - Tekton Pipelines: v0.22.0
-    - Tekton Triggers: v0.12.1
-    - ClusterTasks based on Tekton Catalog 0.22
+
+    - Tekton Pipelines: v0.28.0
+    - Tekton Triggers: v0.16.0
+    - ClusterTasks based on Tekton Catalog 0.28
```

Comment:
How do we ensure that's the case ? and up-to-date ?

Reply:
great question, 🧑‍💻 
im have few ideas, i shall take care of this in the next iteration, as it is a bit tricky as the dependencies are pulled by a script now.

I will capture this in an improvement story.